Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1033,32 +1034,38 @@ public void Restart()
             return compare;
         }
 
-        /// <summary>
-        /// Only loads the latest version of each assembly based upon the folder name.
-        /// eg. MyAssembly_11.9.3.6  - will be ignored.
-        ///     MyAssembly_12.2.3.6  - will be loaded.
-        /// </summary>
-        /// <param name="path">Path to enumerate.</param>
-        /// <param name="cleanup">Set to true, to try and clean up earlier versions.</param>
-        /// <returns>IEnumerable{string} of filenames.</returns>
-        protected IEnumerable<string> GetLatestDLLVersion(string path, bool cleanup = true)
+        private IEnumerable<string> GetPlugins(string path, bool cleanup = true)
         {
             var dllList = new List<string>();
             var versions = new List<(Version PluginVersion, string Name, string Path)>();
             var directories = Directory.EnumerateDirectories(path, "*.*", SearchOption.TopDirectoryOnly);
 
+            var serializer = new JsonSerializer();
```

Comment:
```suggestion
```

There's already a `_jsonSerializer` in this instance.

Reply:
Done a search and can't find it in this version of the code. Can see an XMLSerializer, but no _jsonSerializer.

Reply:
Right, I was looking in the wrong file 🙃 
You can pull in a jsonSerializer the same way the xmlSerializer is initialized